gen_host_status.yml

```diff
@@ -0,0 +1,38 @@
+- hosts: all
+  gather_facts: false
+  tasks:
+    - name: A debug msg all hosts will show except for skipped ones
+      debug:
+        msg: "Playing {{ ansible_host }}"
+      when: "'_skipped' not in ansible_host"
+
+    - name: Hosts haven't really changed, but we will say they have
+      debug:
+        msg: "I am a changed host."
+      changed_when: my_status == 'changed'
```

Comment:
What's going on with `my_status` here?

Looks like you moved to an approach of testing `ansible_host` so maybe this variable is no longer needed.

Reply:
Yeah...a bug